Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -294,14 +299,16 @@ view push state =
                   , background Color.transparent
                   , accessibility if any (_ == state.props.currentStage) [RideAccepted, RideStarted, HomeScreen] && not isAnyOverlayEnabled state then ENABLE else DISABLE
                   , accessibilityHint $ camelCaseToSentenceCase (show state.props.currentStage)
-                  ][
-                  linearLayout
-                    [ height if any (_ == state.props.currentStage) [RideAccepted, RideStarted, ChatWithDriver] && os /= "IOS" then (V (((screenHeight unit)/ 15)*10)) else MATCH_PARENT
-                    , width MATCH_PARENT
-                    , accessibility DISABLE_DESCENDANT
-                    , id (getNewIDWithTag "CustomerHomeScreenMap")
-                    , visibility if state.props.isSrcServiceable then VISIBLE else GONE
-                    ][]
+                  ][ 
+                    if isHomeScreenView state then homeScreenView push state else emptyTextView state
```

Comment:
@aryan-13 append the view to the linearLayout as an array and avoid using emptyTextView